### DISCUSSION TITLE
How to convert DeepARModel(TimeSeriesPredictor) to an onnx model?

### DISCUSSION THREAD

**Discussion by jmren168** [autogluon/autogluon#3395] [Q&A]
Hi there,

I'm trying to convert a TimeSeriesPredictor (i.e., DeepARModel) to onnx format, but don't have any clues. If anyone could give some hints, I would highly appreciated. 

Best,

**Comment by shchur** [comment]
Hi, we currently do not offer such functionality. I recommend asking this question to the GluonTS maintainers. There is an open issue on this topic https://github.com/awslabs/gluonts/issues/726 but, unfortunately, I don't know what the current status is.

**Reply by jmren168** [reply]
Thanks for the quick reply. I'll try your recommendation
